readme.md

```diff
@@ -77,6 +77,40 @@ Note: This does not cancel promises that are already running.
 
 Get or set the concurrency limit.
 
+### limitFunction(fn, options)
+
+Return a function with limited concurrency.
+
+Note: Unlike fucntion return by `pLimit`, the return by `limitFunction` is already limited, do not need call with `limit`.
```

Comment:
Multiple typos and does not make it super clear what the difference is. Maybe try using ChatGPT.

Reply:
Should be in index.d.ts too